### DISCUSSION TITLE
Problemas con los modelos

### DISCUSSION THREAD

**Discussion by Alejandro121499** [spatie/laravel-permission#1849] [Q&A]
Porque cuando lo publico no me salen los modelos, eso hace que tenga problemas para crear los CRUD que hasta ahora no he podido solucionar, gracias 
![image](https://user-images.githubusercontent.com/43188408/132456110-ea50b4b1-1572-400c-859d-6990e11872a6.png)

**Comment by erikn69** [comment]
En primer lugar no puedes preguntar en español, solo en ingles, second you didn't read the docs, it is just a laravel relationship, you don´t need `role_has_permissions`, only `$role->permissions()->get();`

**CRUD EXAMPLE:** [roles-permissions-laravel](https://github.com/saqueib/roles-permissions-laravel/blob/master/app/Http/Controllers/RoleController.php)

**Comment by neoacevedo** [comment]
Y la regla donde dice que no se puede preguntar en español, dónde está?

**Reply by erikn69** [reply]
No hay ninguna, pero si lo haces no vas a recibir respuesta de los autores, entonces deseas quedarte sin respuesta a tu problema?

**Comment by neoacevedo** [comment]
Ya entendí la pregunta. Simplemente es ubicar el cursor sobre la palabra Role o Permission, Ctrl+Espacio y VSCode muestra una lista. De esa lista, seleccionar el que hereda de Models: `Spatie\Permissions\Model` y el IDE hará el import en la parte superior como `use Spatie\Permission\Models\Permission;`:

![imagen](https://user-images.githubusercontent.com/438372/134388414-d485a908-8606-4d0d-80bb-0e5a8d1625a8.png)
